Assignment/Program.cs

```diff
@@ -1,7 +1,19 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿int reverse = 0;

```

Comment:
This can be inside the while loop just after int number = originalNumber;

Reply:
ok